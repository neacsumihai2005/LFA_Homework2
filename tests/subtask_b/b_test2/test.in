8
0 1 2 3 4 5 6 7
16
0 1 a
0 5 b
1 6 a
1 2 b
2 0 a
2 2 b
3 2 a
3 6 b
4 7 a
4 5 b
5 2 a
5 6 b
6 6 a
6 4 b
7 6 a
7 2 b
0
1
2
