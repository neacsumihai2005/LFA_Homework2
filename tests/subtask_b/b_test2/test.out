0 -> q0
1 -> q1
2 -> q2
3 -> q3
4 -> q4
5 -> q5
6 -> q6
7 -> q7

{q0, q4}: ({q1, q7}, a) ({q3, q5}, b) 
{q6}: ({q6}, a) ({q0, q4}, b) 
{q1, q7}: ({q6}, a) ({q2}, b) 
{q3, q5}: ({q2}, a) ({q6}, b) 
{q2}: ({q0, q4}, a) ({q2}, b) 
